overrides/home.html

```diff
@@ -28,6 +28,9 @@
         <a href="https://pretalx.com/pytexas-2024/" class="pytx-button pytx-button--primary">
             Submit to our CFP
         </a>
+        <a href="https://pretix.eu/pytexas/2024/" class="pytx-button pytx-button--primary">
+          Purchase Tickets
+      </a>
```

Comment:
This is nitpicky AF, so feel free to ignore, but the end tag isn't aligned with the start tag. It's a small thing, but could cause a misunderstanding sometime later.

Reply:
Fixed